web_app/templates/web_app/sysadmin/user_activation_email.txt

```diff
@@ -0,0 +1,9 @@
+Hi {{ user.first_name }} {{ user.last_name }},
```

Comment:
This file seems redundant with user_activation_email.html; is it auto-generated?  If so, I would propose .gitignore-ing it.

Reply:
This was intentional. Not too knowledgeable on e-mail content types but it seems like sending a plain text version and an HTML version is a good idea in case someone can not render HTML on their client. 
https://docs.djangoproject.com/en/3.1/topics/email/#sending-alternative-content-types